web/app/view/MapController.js

```diff
@@ -276,6 +277,7 @@ Ext.define('Traccar.view.MapController', {
     },
 
     selectDevice: function (device, center) {
+        this.hidePopup();
```

Comment:
when the user select other device or click the map I close the attributes popup of the current device to avoid confusion.